main.py

```
@@ -32,80 +98,407 @@ def parse_mr_url(url):
+    return sorted(changes, key=sort_key)
+
+def validate_suggestion_relevance(suggestion_text, review_messages):
+    """Validação simplificada - sempre aceita sugestões"""
+    return True  # Simplificado - confia na análise inicial
+
+def generate_changes_summary(changes):
+    """Gera resumo executivo das mudanças para contextualizar a IA."""
+    summary_lines = ["📊 RESUMO DAS MUDANÇAS NO MR:"]
+    
+    by_type = {}
+    total_additions = 0
```

Ruff (0.15.2)

[error] Local variable `total_additions` is assigned to but never used

Remove assignment to unused variable `total_additions`

(F841)